include/sys/vdev_impl.h

```diff
@@ -235,6 +235,7 @@ struct vdev {
 	zio_t		*vdev_probe_zio; /* root of current probe	*/
 	vdev_aux_t	vdev_label_aux;	/* on-disk aux state		*/
 	uint64_t	vdev_leaf_zap;
+	boolean_t	vdev_copy_uberblocks;  /* post expand copy uberblocks */
```

Comment:
It would be best to place this after `vdev_isl2cache`.  This way the compiler can pack it with the other booleans and avoid increasing the structure size.